src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities.mdx

```diff
@@ -0,0 +1,275 @@
+---
+title: User management capabilities 
+tags:
+  - Accounts
+  - New Relic One user management
+metaDescription: An explanation of New Relic user capabilities and what they govern. 
+redirects:
+---
+
+import newRelicUserCapabilitiesUi from 'images/user-management-capabilities.png'
+
+A role, whether one of our [pre-built roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles) or a custom role, is a set of capabilities. You can think of a "capability" as a granular thing a New Relic user can do. Examples of capabilities are: the capability to view APM app settings, or modify alert conditions, or manage data retention settings. 
```

Comment:
Is there a limit on how many capabilities one role has? Example: An org has 2 teams, Team A and B. Team A has one role with X capabilities, but team B has a separate role with Y capabilities?

Could be helpful to give an example situation in 1-2 sentences that makes concrete what this means in practice.